misc/testerina/modules/testerina-runtime/src/main/java/org/ballerinalang/test/runtime/BTestRunner.java

```diff
@@ -575,7 +578,11 @@ private Path getConfigPath(TestSuite testSuite) {
         if (!moduleName.equals("")) {
             configFilePath = configFilePath.resolve(ProjectConstants.MODULES_ROOT).resolve(moduleName);
         }
-        return configFilePath.resolve(ProjectConstants.TEST_DIR_NAME);
+        configFilePath = configFilePath.resolve(ProjectConstants.TEST_DIR_NAME).resolve(CONFIG_FILE_NAME);
+        if (!Files.exists(configFilePath)) {
+            configFilePath = Paths.get(RuntimeUtils.USER_DIR).resolve(CONFIG_FILE_NAME);
```

Comment:
What can happen if both test directory and project does not have config.toml

Reply:
It would give an error at variable initialization.
If mandatory variable - "value not found" error
If optional - use the default value